tutorials/solution_strategies.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>These comments are fair, but also warrant a remark at the beginning about this being work in progress.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>